dev-libs/libstrophe/libstrophe-0.9.2.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A simple, lightweight C library for writing XMPP clients"
+HOMEPAGE="http://strophe.im/libstrophe"
+SRC_URI="https://github.com/strophe/libstrophe/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libxml2 +tls doc"
```

Comment:
Switch `tls` to `ssl`. The description says "Add support for SSL/TLS connections (Secure Socket Layer / Transport Layer Security)"

I would switch libxml2 to expat, because usage of libxml2 is usually the default. Then you don't have to include any of the USE flags in metadata.xml file, because `doc expat ssl` are globally defined.

Also keep in alphabetical order, ie doc expat ssl.